crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -1295,6 +1340,47 @@ impl TryFrom<CertifiedTransaction> for SuiCertifiedTransaction {
     }
 }
 
+/// The certified Transaction Effects which has signatures from >= 2/3 of validators
+#[derive(Clone, Debug, Serialize, Deserialize, JsonSchema)]
+#[serde(rename = "CertifiedTransactionEffects", rename_all = "camelCase")]
+pub struct SuiCertifiedTransactionEffects {
+    pub transaction_effects_digest: TransactionEffectsDigest,
+    pub effects: SuiTransactionEffects,
+    /// authority signature information signed by the quorum of the validators.
+    pub auth_sign_info: AuthorityStrongQuorumSignInfo,
```

Comment:
the `transaction_effects_digest` and `auth_sign_info` duplicate the fields in `SuiCertifiedTransaction`, which is also part of `SuiExecuteTransactionResponse`. Is there any reason why we need these here?

Reply:
there's a TODO `TODO: Change to CertifiedTransactionEffects eventually.`  in line 364.